src/topology/continuous_map.lean

```diff
@@ -44,20 +44,17 @@ instance [inhabited β] : inhabited C(α, β) :=
 lemma coe_inj ⦃f g : C(α, β)⦄ (h : (f : α → β) = g) : f = g :=
 by cases f; cases g; cases h; refl
 
-/--
-The identity as a continuous map.
--/
-def id : C(α, α) := ⟨id⟩
+/-- The identity as a continuous map. -/
+def id : C(α, α) := ⟨id, continuous_id⟩
```

Comment:
can you explain why a change is needed here?

Reply:
Well I don't remember but if it works both ways I probably did a merge wrong because I do not recall changing this file on purpose... I will check which merge went wrong and fix it everywhere